docs/docs/07-cli.md

```diff
@@ -417,9 +417,9 @@ Produced source code can be regenerated by running a command again and is not me
 
 * [ignite](#ignite)	 - Ignite CLI offers everything you need to scaffold, test, build, and launch your blockchain
 * [ignite generate dart](#ignite-generate-dart)	 - Generate a Dart client
-* [ignite generate openapi](#ignite-generate-openapi)	 - Generate generates an OpenAPI spec for your chain from your config.yml
-* [ignite generate proto-go](#ignite-generate-proto-go)	 - Generate proto based Go code needed for the app's source code
-* [ignite generate vuex](#ignite-generate-vuex)	 - Generate Vuex store for you chain's frontend from your config.yml
+* [ignite generate openapi](#ignite-generate-openapi)	 - Generate an OpenAPI spec for your chain from your config.yml
```

Comment:
This file is auto generated, can we revert the changes?

Reply:
oh hadnt realised

Reply:
if it's going to be auto-generated why do we need to revert?